### PR TITLE
ROX-10245: Correct policy naming 

### PR DESCRIPTION
**Pull request by vikin91**
## Description

This PR changes the passive voice of system policies for network policies: `MissingIngressNetworkPolicy == true` into `HasIngressNetworkPolicy == false` in the `augmentedobjs.NetworkPoliciesApplied` object. All other fields containing the word _missing_ were updated accordingly to contain the word _has_.

## Checklist
- [x] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- ~~[ ] Evaluated and added CHANGELOG entry if required~~
    - There was no entry so far 
- ~~[ ] Determined and documented upgrade steps~~
    - Not required
- ~~[ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~
   - Shall be done in [ROX-10249](https://issues.redhat.com/browse/ROX-10249)

## Testing Performed

- [x] CI
- [x] Manually deployed a local cluster and tested various policies combining: `[missing|has] [ingress|egress] network policy`

### CODE REVIEW

**Review by lvalerom — APPROVED** [review]
Looks good. If the CI doesn't complain I think it's good to go.

File: central/policy/service/validator_test.go [hunk 1]
```diff
@@ -168,12 +168,12 @@ func (s *PolicyValidatorTestSuite) TestValidateDescription() {
...
-			the spring of hope, it was the winter of despair, we had everything before us, we had nothing before us, we were all 
-			going direct to Heaven, we were all going direct the other way--in short, the period was so far like the present 
-			period that some of its noisiest authorities insisted on its being received, for good or for evil, in the superlative 
+			feet after that tornado ripped their town to shreds and left them nothing but pineapple and gum.  It was the It was
+			the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the
+			epoch of belief, it was the epoch of incredulity, it was the season of Light, it was the season of Darkness, it was
+			the spring of hope, it was the winter of despair, we had everything before us, we had nothing before us, we were all
+			going direct to Heaven, we were all going direct the other way--in short, the period was so far like the present
+			period that some of its noisiest authorities insisted on its being received, for good or for evil, in the superlative
```
Comment (lvalerom) [inline central/policy/service/validator_test.go:176]: Just checking if you change this on purpose.

Comment (lvalerom) [reply central/policy/service/validator_test.go:176]: I don't think it matters. Judging by the code the test checks if the `Policy Description` has more than 800 chars.

**Review by md2119 — APPROVED** [review]
Is the name change primarily for the bi-directionality support?

**Comment by vikin91** [comment]
@md2119 Yes. By enabling the possibility to react on positive (policy exists) and negative (policy missing) we though that if would be easier for the users to use the following:
- Has Ingres policy: true, false
instead of the previous version:
- Missing Ingress policy: true, false

**Comment by vikin91** [comment]
@fredrb do you want to give it a quick look maybe and check whether I captured the goals of the ticket exactly as planned?

**Review by fredrb — APPROVED** [review]
All good from my side, thanks @vikin91!

**Pull request merged**